pages/home.md

```diff
@@ -41,34 +42,49 @@ permalink: /
               <img src="{{site.baseurl}}/assets/img/solid-pod-tour.svg" alt="[]" />
             </figure>
           </div>
+
           <div class="column is-offset-1 is-5">
-            <h2 class="title">All of your data, under your control</h2>
+            <h2 class="title">Solid</h2>
             <p class="content is-large">
-            <a href="{{site.baseurl}}/about">Solid</a> lets people store their 
-            data securely in decentralized data stores 
-            called <b>Pods</b>. Pods are like secure personal web servers for data.
-            All data in a pod is accessible via the 
-            <a href="https://solid.github.io/specification/" target="_blank">Solid Protocol</a>.
-            When data is stored in someone's 
-            pod, they control who and what can access it.</p>
-                                
+
+              <a href="{{site.baseurl}}/about">Solid</a> is a project that envisions an ecosystem where 
+              people securely store their data in Pods, and applications can 
+              access data from any Pod if they have 
+              been granted access.
+            </p>
             <p class="content is-large">
-            Solid is led by the inventor of the Web, 
-            <b><a href="{{site.baseurl}}/team">Sir Tim Berners-Lee</a></b>, 
-            to help realise his vision 
-            for its future.</p>
-            </div>
+              Underlying the ecosystem are the Solid specifications. The
```

Comment:
We could move the Underlying paragraph out to a different section - but I wanted to give a general idea of what I mean by having some definition of what Solid is on the first page.